src/round_tracking.py

```diff
@@ -0,0 +1,86 @@
+# Adam's attempt at a data collection system as another tool in the playtesting
+# arsenal.
+#
+# Basic idea is to run multiple enemy-less rounds with a set of companions,
```

Comment:
How are you envisioning the player recording that data? Playing out each turn manually and then enter the numbers?